ECMA2Yaml/ECMA2Yaml/Nuget/ECMA2Yaml.nuspec

```diff
@@ -15,6 +15,7 @@
   <files>
     <file src="..\bin\Release\*.*" target="tools" exclude="..\bin\Release\*.xml;..\bin\Release\*.pdb;..\bin\Release\*.txt;..\bin\Release\log.*"/>
     <file src="..\..\IntellisenseFileGen\bin\Release\IntellisenseFileGen.*" target="tools"/>
+    <file src="..\..\MSDNUrlPatch\bin\Release\MSDNUrlPatch.*" target="tools"/>
```

Comment:
please update the signing filter in https://github.com/docascode/ECMA2Yaml/blob/develop/azure-pipelines.yml#L59